### ISSUE TITLE
Paused jobs not properly handled by queue state url handler

### ISSUE THREAD

**Issue by mshwery** [bee-queue/arena#132]
Because `paused` is not in the [accepted list of bull queue states](https://github.com/bee-queue/arena/blob/master/src/server/views/helpers/queueHelpers.js#L27), it is treated as [an individual job](https://github.com/bee-queue/arena/blob/cbfa82ac09c12ae074a5fe8c23d52c64ee178de2/src/server/views/dashboard/index.js#L10-L11) which fails on `hgetall` with a `WRONGTYPE` error from Redis:

> WRONGTYPE Operation against a key holding the wrong kind of value

**Comment by mshwery** [comment]
Wondering if it makes sense to map to all of these states in particular: https://github.com/OptimalBits/bull/blob/5e52e045a0a58d5126f4af3e4572b4af86cc7f9d/lib/getters.js#L20-L30

**Comment by mshwery** [comment]
Addressed in #133

**Comment by fbukevin** [comment]
I still got this problem.

**Comment by mshwery** [comment]
Yeah I think the original PR didn't fully address the issue, just pushed it deeper.
